tools/dxexp/dxexp.cpp

```diff
@@ -296,7 +296,7 @@ static HRESULT PrintAdapters() {
     hr = e.m_hr;
     json_printf("%c { \"err\": \"unable to print information for adapters - "
                 "0x%08x\" }\n",
-                comma, hr);
+                comma, (unsinged int)hr);
```

Comment:
need to be unsigned

But doesn't have access to push fix:(

Reply:
Fixed.